gateway-js/src/index.ts

```diff
@@ -1312,6 +1110,13 @@ export {
   GatewayConfig,
   ServiceEndpointDefinition,
   CompositionInfo,
+  IntrospectAndCompose,
 };
 
 export * from './datasources';
+
+export {
+  SupergraphSdlUpdateFunction,
+  SubgraphHealthCheckFunction,
```

Comment:
Should be exporting `SupergraphManager` and `GetDataSourceFunction` types as well

Reply:
c546ce50